pkg/objstore/objstore.go

```diff
@@ -275,6 +279,10 @@ func (b *metricBucket) Delete(ctx context.Context, name string) error {
 	return err
 }
 
+func (b *metricBucket) IsObjNotFoundErr(err error) bool {
+	return b.IsObjNotFoundErr(err)
```

Comment:
Little confused here ... does this not just call itself?

Reply:
should it be `b.bkt.IsObjNotFoundErr(err)`?

Reply:
yup, nice catch, thanks!